boost-maven/boost-boms/boost-ee8-apis-bom/pom.xml

```diff
@@ -46,6 +46,11 @@
 				<artifactId>javax.json-api</artifactId>
 				<version>1.1</version>
 			</dependency>
+			<dependency>
+				<groupId>javax.json.bind</groupId>
+				<artifactId>javax.json.bind-api</artifactId>
+				<version>1.0</version>
+			</dependency>
 		</dependencies>
```

Comment:
Should we start by pulling in the EE 8 aggregate dependency (same thought for EE 7 and MPs)?   We could always override I believe.

Reply:
@scottkurz  - do you mean this?

<dependency>
    <groupId>javax</groupId>
    <artifactId>javaee-api</artifactId>
    <version>8.0</version>
</dependency>